### ISSUE TITLE
Enhancement: Clarify https://github.com/Eflores89/proyectos/tree/master/data/country_clusters

### ISSUE THREAD

**Issue by mexindian** [Eflores89/proyectos#1]
Estaba pensando, se le podria agregar un poco de color a la hierarchy, creo q `ggdendro()` lo permite, no? Los colores podrian representar por ejemplo los continentes o subcontinentes.

O sino, otra cosa q podria ser cool es meter todos estos nodos en un mapa. Segun veo hay 2 "equipos", que se subdividen en 2 equipos mas, y etc etc. Podria ser cool meter la primera bifurcacion como azul/rojo (o algo mas politicamente neutral como naranja y violeta... jajaja). Y luego alomejor asignar 8 categorias de naranja y de azul dentro de estos y asi tener un mapa donde se pueden ver los equipos principales y al menos dos niveles de desagregacion dentro de cada uno. Que tal te gusta?

**Comment by Eflores89** [comment]
Si... de hecho pensé en un mapa, sabes de algún paquete dónde lo pueda hacer rápidamente? Me da flojera aventarme toda la homologación de los países, etc...

**Comment by mexindian** [comment]
no... pos si. he utilizado `leaflet` para hacer mapas, pero con cordenadas solamente... no se si.......
jajajja q vaciado.... estaba buscando un tutorial q estaba casi seguro q habia visto uno... y mira lo q encontre. SEGURO te cae como anillo al dedo: https://rpubs.com/walkerke/leaflet_choropleth :D
